misc/config_tools/schema/checks/rdt_support.xsd

```diff
@@ -71,31 +71,33 @@
                   )
                   ">
     <xs:annotation>
-      <xs:documentation>if RDT_ENABLED is 'y', there should not be any CLOS IDs overlap between a vCAT VM and any other VMs</xs:documentation>
+      <xs:documentation>When hypervisor Intel Resource Director Technology is enabled, CLOS IDs may not overlap between any VMs.</xs:documentation>
     </xs:annotation>
   </xs:assert>
 
   <xs:assert test="every $vm in //vm satisfies $vm//load_order != 'SERVICE_VM' or count($vm//lapic_passthrough[text() = 'y']) = 0 or count(//nested_virtualization_support[text() = 'y']) > 0">
     <xs:annotation>
-      <xs:documentation>Service VM cannot use LAPIC passthrough unless GUEST_FLAG_NVMX_ENABLED is set.</xs:documentation>
+      <xs:documentation>The Service VM may not use LAPIC passthrough unless hypervisor Nested Virtualization is enabled.</xs:documentation>
     </xs:annotation>
   </xs:assert>
 
-  <xs:assert test="not (//hv//RDT/RDT_ENABLED = 'y' and //hv//SSRAM/SSRAM_ENABLED = 'y')"/>
+  <xs:assert test="not (//hv//RDT/RDT_ENABLED = 'y' and //hv//SSRAM/SSRAM_ENABLED = 'y')">
+    <xs:annotation>
+      <xs:documentation>The hypervisor Intel Resource Director Technology and Software SRAM settings may not be enabled at the same time.</xs:documentation>
+    </xs:annotation>
+  </xs:assert>
 
   <xs:assert test="hv//SSRAM_ENABLED = 'n' or empty(vm[load_order ='PRE_LAUNCHED_VM' and vm_type='RTVM']) or
 		   every $cap in caches/cache[@level=3]/capability[@id='Software SRAM'] satisfies
 		   (compare($cap/end, '0x80000000') &lt; 0 or compare($cap/start, '0xf8000000') &gt;= 0)">
     <xs:annotation acrn:severity="warning">
-      <xs:documentation>The physical software SRAM region shall not overlap with pre-defined regions in guest.
+      <xs:documentation>The physical software SRAM region may not overlap with pre-defined regions in any VM.
 
 When a pre-launched RT VM is enabled, the physical software SRAM is allocated to it at the same guest physical
 address. Thus it is assumed that the software SRAM region does not overlap with any pre-defined region in the
-pre-launched VM, such as the guest PCI hole which resides at 2G - 3.5G.
-
-This error cannot be fixed by tweaking the configurations. Report to _GitHub:
-https://github.com/projectacrn/acrn-hypervisor/issues if you meet this.</xs:documentation>
+pre-launched VM, such as the guest PCI hole which resides at 2GB - 3.5GB.
```

Comment:
Change all mentions of "RT VM" to "RTVM" per ACRN glossary (or spell out real-time for clarity)? https://projectacrn.github.io/latest/glossary.html#term-RTVM